test/test

```diff
@@ -577,32 +577,6 @@ checkoutput <<EOF
 test.log 0 zero
 EOF
 
-cleanup 18
```

Comment:
You are removing an existing test-case without any comment.  Are you going to break the scenario covered by the test-case without telling potential users about the breakage?